README.md

```diff
@@ -26,6 +26,7 @@ The Edge AI Suites project hosts a collection of sample applications organized a
 | Metro AI Suite | [Loitering Detection](metro-ai-suite/loitering-detection) | [Link](metro-ai-suite/loitering-detection/docs/user-guide/get-started.md) | [Customize the Application](metro-ai-suite/loitering-detection/docs/user-guide/how-to-customize-application.md) |
 | Metro AI Suite | [Image-Based Video Search](metro-ai-suite/image-based-video-search) | [Link](metro-ai-suite/image-based-video-search/docs/user-guide/get-started.md) | [Build](metro-ai-suite/image-based-video-search/docs/user-guide/how-to-build-source.md) and [Deployment](metro-ai-suite/image-based-video-search/docs/user-guide/how-to-deploy-helm.md) instructions |
 | Metro AI Suite | [Smart Parking](metro-ai-suite/smart-parking) | [Link](metro-ai-suite/smart-parking/docs/user-guide/get-started.md) | [Customize the Application](metro-ai-suite/smart-parking/docs/user-guide/how-to-customize-application.md) |
+| Metro AI Suite | [Smart Intersection](metro-ai-suite/smart-intersection) | [Link](metro-ai-suite/smart-intersection/docs/user-guide/get-started.md) | [Deployment](metro-ai-suite/smart-intersection/docs/user-guide/how-to-deploy-docker.md) |
```

Comment:
Please move up so that the table rows is sorted alphabetically.

Reply:
Sorted them alphabetically in latest commits